services/agents/rds_test.go

```diff
@@ -90,36 +90,36 @@ func TestRDSExporterConfig(t *testing.T) {
 			`config`: strings.TrimSpace(`
 ---
 instances:
-- region: us-east-1
```

Comment:
does this means that this format will not work?

Reply:
I'm woried about backward compatibility

Reply:
It will work, but default formatting rules are changed. In tests we use simple assertion for strings equality, so I formated hardcoded yamls it tests.

For example SaaS returns STT checks formated with `yaml.v3` and before that PR managed unmarshalled them with `yaml.v2`